packages/daimo-common/test/keySlot.test.ts

```diff
@@ -0,0 +1,37 @@
+import assert from "node:assert";
+import test from "node:test";
+
+import { SlotType, findUnusedSlot, getSlotLabel } from "../src/keySlot";
+
+const testCases: [number, string][] = [
+  [0, "Mobile A"],
+  [25, "Mobile Z"],
+  [26, "Mobile AA"],
```

Comment:
add `Computer A`

Reply:
so we'd have `Phone A` / `Computer A` / `Passkey A` or `Passkey Backup A` 

lmk if that looks right

Reply:
done in #383